elixir/apps/domain/lib/domain/auth/adapters/google_workspace/api_client.ex

```diff
@@ -196,6 +196,7 @@ defmodule Domain.Auth.Adapters.GoogleWorkspace.APIClient do
   # stop.
   #
   # For members, this happens quite often and we want to return an empty list.
+  # For organization units, this is expected if the Google account has no org units.
```

Comment:
If this is true for Org Units, would it stand to reason that this is also true for Groups?  If so, how would we distinguish between a Google Workspace Account having no groups and Google's API giving us bad data like we've seen in the past?

Reply:
Maybe, and we might hit that error in the future, but the fact we haven't yet should give us some reassurance.

It would be better to crash than assume an empty group list and trigger the circuit breaker I would say.

Reply:
All good points.  Hopefully we won't ever run in to this with groups.